packages/cli/package.json

```diff
@@ -176,10 +178,13 @@
         "isolatedModules": true
       }
     },
+    "setupFilesAfterEnv": [
+      "@alex_neo/jest-expect-message"
+    ],
     "verbose": false,
     "testEnvironment": "node",
     "testMatch": [
-      "<rootDir>/test/**/*.test.ts"
+      "<rootDir>/test/commands/pull.test.ts"
```

Comment:
This seems like its going to skip tests

Reply:
Oh, that was for debugging. Thanks for catching that. I need to remove it.

Reply:
In the future, you can use `jest -t 'some test name'` or more likely `yarn test-unit -t 'some test name'` to run a specific test instead of changing config

Reply:
I was doing that locally, but I wanted to run only this test file in CI to see if there was some test pollution